L1Trigger/Phase2L1ParticleFlow/python/l1pfJetMet_cff.py

```diff
@@ -30,7 +32,8 @@
 
 l1PFJets = cms.Sequence(
     ak4PFL1Calo + ak4PFL1PF + ak4PFL1Puppi +
-    ak4PFL1CaloCorrected + ak4PFL1PFCorrected + ak4PFL1PuppiCorrected
+    ak4PFL1CaloCorrected + ak4PFL1PFCorrected + ak4PFL1PuppiCorrected +
+    L1SeedConeJetProducer
 )
```

Comment:
also for the SeedConeJet it could make sense to have both the raw and the corrected one (but maybe this we can do later when preparing the PR for the Menu group)

Reply:
I modified the config file now to compute the SeedConeJet for PF and Puppi, but haven't added the corrections yet